Shared/Samples/Authenticate with OAuth/AuthenticateWithOAuthView.swift

```diff
@@ -0,0 +1,96 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+import ArcGISToolkit
+
+struct AuthenticateWithOAuthView: View {
+    /// The authenticator to handle authentication challenges.
+    @ObservedObject var authenticator: Authenticator
+
+    init() {
+        // Create an authenticator.
```

Comment:
```suggestion
        // Creates an authenticator.
```
I believe it's a standard to use an active voice for the comments